src/app/shared/services/children/children.service.ts

```diff
@@ -21,7 +21,7 @@ export class ChildrenService {
         params = params.set('SearchString', parameters.searchString);
       }
 
-      if (parameters.isParent !== undefined) {
+      if (parameters.isParent !== null) {
```

Comment:
It may be better to leave both conditions here because if the value is `undefined`, the same error will occur
At least in [one place](https://github.com/ita-social-projects/OoS-Frontend/blob/c739f8c35c9a3a285f2489b98edecba062533ed0/src/app/shell/admin-tools/data/users/users.component.ts#L94), we assign `undefined` to `isParent`